package.json

```diff
@@ -2,9 +2,9 @@
   "name": "nimble-styling",
   "devDependencies": {
     "@types/color": "^3.0.0",
+    "@types/react": "16.7.7",
+    "@types/react-native": "0.57.4",
```

Comment:
Were these changes required for your project to work?

Reply:
No, i don't know why package.json has changes in it. I'll checkout the package.json from develop.